app/gui/controller/engine-protocol/src/language_server/types.rs

```diff
@@ -249,6 +251,13 @@ pub enum ExpressionUpdatePayload {
     },
 }
 
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
+#[allow(missing_docs)]
+pub struct Warnings {
+    pub count: usize,
+    pub value: Option<String>,
+}
```

Comment:
Please copy the docs from [protocol specification](https://github.com/enso-org/enso/blob/f73f24362b8e24b67b928ef7035ddfac77d1b721/docs/language-server/protocol-language-server.md) about this structure, including fields (as the `value` field is not intuitive).

Reply:
Done